minio/1.11/README.md

```diff
@@ -0,0 +1,160 @@
+
+Minio server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL.
+
+This installation uses the single node version of Minio (Minio FS), checkout the [Minio docs](https://docs.minio.io) for more details.
+
+The instructions below use a pinned hostname constraint to ensure the application is always restarted on the same host by Marathon. This allows it to get back to its data but means that you could lose data if that agent goes down.
+
+- Estimated time for completion: less than 5 minutes.
+- Target audience:
+    - Operators
+    - Application admins
+    - Developers
+    - Devops Engineers
+- Scope: Learn to install Minio on DC/OS and learn to use it using Minio command line tool `mc`.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Minio Installation using GUI](#minio-installation-using-gui)
+- [Customize Minio using GUI](#customize-minio-using-gui)
+- [Minio Installation using CLI](#minio-installation-using-cli)
+- [Access Minio Browser](#access-minio-browser)
+- [Install Minio Client](#install-minio-client)
+- [Configure Minio Client](#configure-minio-client)
+- [Uninstall Minio using CLI](#uninstall-minio-using-cli)
+- [Explore Further](#explore-further)
+
+## Prerequisites
+
+- DC/OS 1.11 or later
+- [Marathon-LB](https://docs.mesosphere.com/services/marathon-lb/) must be installed and running
+- Identify [IP of the public agent](https://docs.mesosphere.com/1.11/administering-clusters/locate-public-agent/) where Marathon-LB or an available hostname configured to point to the public agent(s) where Marathon-LB is running.
+
+## Minio installation using GUI
+
+- Login to DC/OS admin, click on Catalog on the left menu bar, search `minio` and select the package with following logo.
+
+![Search & Install Minio](img/search-install.png)
+
+- Click on the `Review & Run` button for the single-click default installation. You can reach your Minio server at `host:9000` where `host` is IP address or hostname of public-agent where Marathon-LB is installed.
+
+![Single-click Install](img/minio-package-install-oneclick.png)
+
+- This installs Minio server instance with factory defaults
+
+- Go to `Services` from left menu bar and verify, minio service status should show `Running` or wait for the status update.
+
+![Minio Service](img/minio-service.png)
+
+- Once Minio has been deployed, navigate to the IP/hostname of Public Agent where Marathon-LB is installed and the port you specified. To customize deployed minio server, continue reading or go to [Access Minio Browser](#access-minio-browser) section.
+
+### Customize Minio using GUI
+
+- To customize the storage and network configuration, select the minio service, then select `Edit`.
```

Comment:
With 1.11 it now opens up the configuration screens into the edit mode by default, so there is no more "Edit" button. Can you re-word this to reflect that?

Reply:
I am talking editing service after deployment, fixed changes, please review them.